### DISCUSSION TITLE
Show IP address of miner in the wallet

### DISCUSSION THREAD

**Discussion by christianloerner** [revoxhere/duino-coin#1830] [Ideas]
Hey Team,

it would be great to see the IP of the miner in the details on the wallet. It is difficult to figure out, which miner belongs to which entry. Even with a given name. This would be great, if you want to upgrade the miner.

Regards
Chris

**Comment by profpandaa** [comment]
The IP would be your network's public IP, which shouldn't differ from miner to miner. Unless, of course, they're running on separate connections.
